.travis.yml

```diff
@@ -15,6 +15,7 @@ before_script:
   - mysql -e 'create database sorcery_test;'
 
 before_install:
+  - gpg --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
   - rvm get stable --auto-dotfiles
```

Comment:
Why are you updating rvm here? AFAIK the `rvm:` above runs before `before_install` and installs Ruby versions before this line is run.

Reply:
It is the case as shown in https://travis-ci.org/Sorcery/sorcery/jobs/478627382
Seems better now, but looks like Rails depends on `bundler < 2`.